### ISSUE TITLE
What do you mean by "time gapped"?

### ISSUE THREAD

**Issue by JazzFan** [szilard/ml-prod#2]
You note that "The real world is non-stationary[;] use a time gapped test set".  What do you mean by "time gapped"?  A test set from a different time, or a test set containing gaps in time, or something else?

**Comment by szilard** [comment]
I mean having e.g. January-March data for training and April data for testing (not random split). Also make sure you are not using any info regarding the training set that becomes available only in April. 

Maybe "time wall" or "time separated" is a better term.
